app/views/case_workers/admin/allocations/_allocation.html.haml

```diff
@@ -1,54 +1,47 @@
-= render partial: 'errors/error_summary', locals: { ep: @allocation, error_summary: t('.error_summary') }
-
 = govuk_notification_banner('Allocation', nil, class: 'govuk-!-display-none') unless !flash.empty?
 
-#api-key.grid-row{ data: { api_key: current_user.api_key } }
-  .column-two-thirds
+#api-key.govuk-grid-row{ data: { api_key: current_user.api_key } }
+  .govuk-grid-column-two-thirds
     = render partial: 'layouts/header', locals: { page_heading: t('.page_heading') }
-  .column-one-third.scheme-filters
-    -# FORM
-    -# AGFS / LGFS Radios
-    = render partial: 'scheme_filters'
 
-= form_tag case_workers_admin_allocations_path, method: :get do
-  = hidden_field_tag :tab, params[:tab]
-  = hidden_field_tag :scheme, params[:scheme]
-  = hidden_field_tag :filter, params[:filter] if params[:filter]
-  = hidden_field_tag :page, params[:page]
-  = hidden_field_tag :value_band_id, params[:value_band_id] if params[:value_band_id]
+= form_with url: case_workers_admin_allocations_path, method: :get, builder: GdsAdpFormBuilder do |f|
+  = f.hidden_field :tab, id: :tab, value: params[:tab]
+  = f.hidden_field :scheme, id: :scheme, value: params[:scheme]
+  = f.hidden_field :filter, id: :filter, value: params[:filter] if params[:filter]
+  = f.hidden_field :page, id: :page, value: params[:page]
+  = f.hidden_field :value_band_id, id: :value_band_id, value: params[:value_band_id] if params[:value_band_id]
+
+  .govuk-grid-row
+    .govuk-grid-column-two-thirds.scheme-filters
+      = render partial: 'scheme_filters', locals: { f: f }
+
+  = f.govuk_fieldset legend: { text: t('.allocate_claims'), size: 'm' } do
+    .govuk-grid-row
+      .govuk-grid-column-one-third
+        = f.govuk_number_field :quantity_to_allocate, label: { text: t('.no_of_claims') }
```

Comment:
was this the fix to the bug i for quantify allocated I found on the first review? Be good to be able to see the changes for fixes - you can also ways squash prior to merging

Reply:
Apologies, it was this line `const quantityToAllocate = $('#quantity-to-allocate-field').val() || false` in the file `app/webpack/javascripts/modules/Modules.AllocationDataTable.js` @jsugarman